images/base/files/etc/systemd/system/kubelet.service.d/10-kubeadm.conf

```diff
@@ -9,4 +9,4 @@ EnvironmentFile=-/var/lib/kubelet/kubeadm-flags.env
 # the .NodeRegistration.KubeletExtraArgs object in the configuration files instead. KUBELET_EXTRA_ARGS should be sourced from this file.
 EnvironmentFile=-/etc/default/kubelet
 ExecStart=
-ExecStart=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_CONFIG_ARGS $KUBELET_KUBEADM_ARGS $KUBELET_EXTRA_ARGS
+ExecStart=/usr/bin/kubelet $KUBELET_KUBECONFIG_ARGS $KUBELET_CONFIG_ARGS $KUBELET_EXTRA_ARGS $KUBELET_KUBEADM_ARGS
```

Comment:
does this match kubeadm upstream? cc @neolit123

Reply:
the NodeRegistrationOptions.KubeletExtraArgs are supposed to be the last user override $KUBELET_EXTRA_ARGS

https://github.com/search?q=repo%3Akubernetes%2Frelease%20KUBELET_CONFIG_ARGS&type=code
`$KUBELET_KUBECONFIG_ARGS $KUBELET_CONFIG_ARGS $KUBELET_KUBEADM_ARGS $KUBELET_EXTRA_ARGS`